.github/workflows/package.yml

```diff
@@ -0,0 +1,97 @@
+name: package
+on: [push, pull_request]
+jobs:
+  linux_wheels:
+    strategy:
+      matrix:
+        arch: [x86, x86_64, arm64, armhf, mips]
+    runs-on: ubuntu-latest
+    container: ghcr.io/frida/x-tools-linux-${{ matrix.arch }}:latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - run: cd agents/tracer && npm install && npm run build
```

Comment:
Also need to build the other agent. Let's just run `make` here instead, which takes care of both.

Reply:
There is no Makefile in there, so I just added the other agent

Reply:
There's one in the root of the repo that takes care of both agents.

Reply:
Done